graph.cpp

```diff
@@ -21,10 +21,16 @@ Graph::Graph(uint64_t num_nodes): num_nodes(num_nodes) {
   }
   num_updates = 0; // REMOVE this later
 
+#ifdef USE_FBT_F
   std::string buffer_loc_prefix = configure_system(); // read the configuration file to configure the system
   // Create buffer tree and start the graphWorkers
   bf = new BufferTree(buffer_loc_prefix, (1<<20), 8, num_nodes, GraphWorker::get_num_groups(), true);
   GraphWorker::start_workers(this, bf);
+#else
+  GraphWorker::set_config(1,1); // TODO: actually get config
```

Comment:
Instead of doing this you should call configure_system

Just ignore the string it returns

Reply:
Sure, I guess we can just pass a nonsense string to the prefix

Reply:
Yeah, we can just leave the path_prefix as the default or even remove it entirely from the configuration file. The system will throw a warning but it won't make an actual difference